tools/cheribsdbox/Makefile

```diff
@@ -295,15 +287,9 @@ CRUNCH_ALIAS_w=	uptime
 CRUNCH_PROGS_usr.bin+=su
 
 # Tracing currently only implemented for CHERI-QEMU architectures
-.if ${MACHINE} == "mips" || ${MACHINE} == "riscv"
+.if ${MACHINE} == "riscv"
 CRUNCH_PROGS_usr.bin+=	qtrace
-.endif
-.if ${MACHINE_ARCH:Mmips64*} || ${MACHINE_ARCH:Mriscv64*}
 CRUNCH_PROGS_usr.bin+=	minimal_count_stats
```

Comment:
This is separate, probably should be another condition

Reply:
Right now they would be duplicate, and we can always split it in the future if another architecture grows support for one of the other of these.

Reply:
I guess though the comment needs to only apply to qtrace

Reply:
Yeah. Moving the comment would also work.